tests/test_butler.py

```diff
@@ -0,0 +1,93 @@
+# This file is part of pipelines_check.
+#
+# Developed for the LSST Data Management System.
+# This product includes software developed by the LSST Project
+# (http://www.lsst.org).
+# See the COPYRIGHT file at the top-level directory of this distribution
+# for details of code ownership.
+#
+# This program is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with this program.  If not, see <http://www.gnu.org/licenses/>.
+
+"""Output butler from pipelines_check test run."""
+
+import os
+import unittest
+
+from lsst.daf.butler import Butler
+
+TESTDIR = os.path.abspath(os.path.dirname(__file__))
+
+MAIN_CHAIN = "demo_collection"
+EXE_CHAIN = "demo_collection_exe"
```

Comment:
add a comment here and in the execution script to make people aware they should stay in sync